From 2a1d16fd1d6729f642d17677bec75f388ce77278 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Mon, 12 Jun 2017 14:57:08 -0400
Subject: [PATCH 2312/2500] drm/amd/display: Guard some DCN-only code in
 dc_debug

Fixing a compile error when building without
CONFIG_DRM_AMD_DC_DCN1_0

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-By: Tom St Denis <tom.stdenis@amd.com>
Acked-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc_debug.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_debug.c b/drivers/gpu/drm/amd/display/dc/core/dc_debug.c
index 263dab6..58402051 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_debug.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_debug.c
@@ -329,6 +329,7 @@ void context_clock_trace(
 	struct core_dc *core_dc = DC_TO_CORE(dc);
 	struct dal_logger *logger =  core_dc->ctx->logger;
 
+#if defined(CONFIG_DRM_AMD_DC_DCN1_0)
 	CLOCK_TRACE("Current: dispclk_khz:%d  dppclk_div:%d  dcfclk_khz:%d\n"
 			"dcfclk_deep_sleep_khz:%d  fclk_khz:%d\n"
 			"dram_ccm_us:%d  min_active_dram_ccm_us:%d\n",
@@ -349,4 +350,5 @@ void context_clock_trace(
 			context->bw.dcn.calc_clk.fclk_khz,
 			context->bw.dcn.calc_clk.dram_ccm_us,
 			context->bw.dcn.calc_clk.min_active_dram_ccm_us);
+#endif
 }
-- 
2.7.4

